### ISSUE TITLE
Need the favicon back  

### ISSUE THREAD

**Issue by kimo** [PySheets/pysheets#83]
Hi @laffra ,

This is Krishna here. Unable to run pysheets (on Intel Mac) unless I manually add the ltk folder into the static folder of the project. Sangram is also facing the same problem. It may be because I am new to the python build process. Please advise. 

```sh
cd <<pysheets>>
cp -r .venv/lib/python3.12/site-packages/ltk src/static
```

Above steps got the project going without errors. Otherwise, i was getting 404's for all **static/ltk/*** files in the below list from the initial landing page. 

    <mpy-config>
        interpreter = ""
        
        [files]
        
    "static/constants.py" = "constants.py"
    "static/menu.py" = "menu.py"
    "static/state.py" = "state.py"
    "static/pysheets.py" = "pysheets.py"
    "static/timeline.py" = "timeline.py"
    "static/editor.py" = "editor.py"
    "static/api.py" = "api.py"
    "static/lsp.py" = "lsp.py"
    "static/preview.py" = "preview.py"
    "static/inventory.py" = "inventory.py"
    "static/models.py" = "models.py"
    "static/selection.py" = "selection.py"
    "static/storage.py" = "storage.py"
    "static/history.py" = "history.py"
    "static/html_maker.py" = "html_maker.py"
    "static/views/spreadsheet.py" = "views/spreadsheet.py"
    "static/views/cell.py" = "views/cell.py"

    **_

> "static/ltk/jquery.py" = "ltk/jquery.py"
>     "static/ltk/widgets.py" = "ltk/widgets.py"
>     "static/ltk/pubsub.py" = "ltk/pubsub.py"
>     "static/ltk/__init__.py" = "ltk/__init__.py"
>     "static/ltk/logger.py" = "ltk/logger.py"
>     "static/ltk/ltk.js" = "ltk/ltk.js"
>     "static/ltk/ltk.css" = "ltk/ltk.css"

_**

    </mpy-config>

**Comment by kimo** [comment]
@kssd fyi.

**Comment by laffra** [comment]
I just noticed that LTK was missing from the MANIFEST.in file, which PyPI uses to bundle up a release. I updated the MANIFEST to include LTK and hosted js/css files. Starting with version 0.2.11, LTK should be included as expected.  This has been tested locally by first running `pip install pysheets-app --upgrade` and then running `pysheets`.

**Comment by kimo** [comment]
Thank you. But we seem to have lost the favicon which is probably not a big deal. But this is what I am getting now.

python code

```
import networkx as nx

graph = nx.DiGraph()

graph.add_edges_from([("root", "a"), ("a", "b"), ("a", "e"), ("b", "c"), ("b", "d"), ("d", "e")])

graph
```

Screenshot

![Screenshot 2024-09-28 at 5 39 38 PM](https://github.com/user-attachments/assets/82ac1397-fc72-469f-ab8b-69035ce621b0)

**Comment by kimo** [comment]
LTK is solved. verified in 3.0.13. We need the favicon back :)

**Comment by kimo** [comment]
Fixed in v0.3.17
